src/constants/fieldMaps.ts

```diff
@@ -172,6 +174,51 @@ export const awsRegions = [
   },
 ];
 
+export const dateFormats = [
+  {
+    value: "MM-dd-yyyy",
+    str: `MM-dd-yyyy - ${getDateCopy("08/31/2022", {
+      dateFormat: "MM-dd-yyyy",
```

Comment:
IMO it may be a bit easier to parse this dropdown if we call this function with `dateOnly` since time format is not being addressed here. But your call!